mad-dog/src/index.js

```diff
@@ -124,6 +152,12 @@ async function generateLibsInstances (numUsers, useZeroIndexedWallet = false) {
   )
 }
 
+// Check to see if verbose mode (print out container logs)
+const isVerbose = () => {
+  return (process.argv[4] && process.argv[4].toLowerCase() === 'verbose') ||
```

Comment:
is this supposed to be the last entry in the `argv` array?

Reply:
yes, that's the plan

Reply:
is there a better way to detemine the last argument here?

Reply:
😸  UPDATING